airtime_mvc/application/controllers/ScheduleController.php

```diff
@@ -297,6 +298,13 @@ public function clearShowAction()
     public function getCurrentPlaylistAction()
     {
         $range = Application_Model_Schedule::GetPlayOrderRangeOld();
+
+        // If there is no current track playing update TuneIn so it doesn't
+        // display outdated metadata
+        if (is_null($range["current"]) && Application_Model_Preference::getTuneinEnabled()) {
+            Application_Common_TuneIn::updateOfflineMetadata();
+        }
```

Comment:
This will spam TuneIn if no track is playing. We call this function every 5 seconds from dashboard.js.
Is there a better way we can catch when Airtime transitions to being off-air?